src/safeds/data/tabular/transformation/_standard_scaler.py

```diff
@@ -0,0 +1,184 @@
+        """
+        Apply the learned transformation to a table.
+
+        The table is not modified.
+
+        Parameters
+        ----------
+        table : Table
+            The table to which the learned transformation is applied.
+
+        Returns
+        -------
+        transformed_table : Table
+            The transformed table.
+
+        Raises
+        ------
+        TransformerNotFittedError
+            If the transformer has not been fitted yet.
+        """
+        # Transformer has not been fitted yet
+        if self._wrapped_transformer is None or self._column_names is None:
+            raise TransformerNotFittedError
+
+        # Input table does not contain all columns used to fit the transformer
+        missing_columns = set(self._column_names) - set(table.column_names)
+        if len(missing_columns) > 0:
+            raise UnknownColumnNameError(list(missing_columns))
+
+        data = table._data.copy()
+        data.columns = table.column_names
+        data[self._column_names] = self._wrapped_transformer.transform(data[self._column_names])
+        return Table._from_pandas_dataframe(data)
+
+    def inverse_transform(self, transformed_table: Table) -> Table:
+        """
+        Undo the learned transformation.
+
+        The table is not modified.
+
+        Parameters
+        ----------
+        transformed_table : Table
+            The table to be transformed back to the original version.
+
+        Returns
+        -------
+        table : Table
+            The original table.
+
+        Raises
+        ------
+        TransformerNotFittedError
+            If the transformer has not been fitted yet.
+        """
+        # Transformer has not been fitted yet
+        if self._wrapped_transformer is None or self._column_names is None:
+            raise TransformerNotFittedError
+
+        data = transformed_table._data.copy()
+        data.columns = transformed_table.column_names
+        data[self._column_names] = self._wrapped_transformer.inverse_transform(data[self._column_names])
+        return Table._from_pandas_dataframe(data)
+
+    def is_fitted(self) -> bool:
+        """
+        Check if the transformer is fitted.
+
+        Returns
+        -------
+        is_fitted : bool
+            Whether the transformer is fitted.
+        """
+        return self._wrapped_transformer is not None
+
+    def get_names_of_added_columns(self) -> list[str]:
+        """
+        Get the names of all new columns that have been added by the StandardScaler.
+
+        Returns
+        -------
+        added_columns : list[str]
+            A list of names of the added columns, ordered as they will appear in the table.
+
+        Raises
+        ------
+        TransformerNotFittedError
+            If the transformer has not been fitted yet.
+        """
+        warnings.warn("StandardScaler only changes data within columns, but does not add any columns.", stacklevel=1)
```

Comment:
I don't think these warnings are useful. Just returning an empty list already expresses that.

Reply:
Removed those warnings (and corresponding tests).  
We also have these warnings currently in the `RangeScaler`, will create a new issue to remove them.

Reply:
Created issue (#323) and already submitted PR for it (#324).